libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -268,7 +268,7 @@ bool ASTJsonConverter::visit(ContractDefinition const& _node)
 {
 	setJsonNode(_node, "ContractDefinition", {
 		make_pair("name", _node.name()),
-		make_pair("documentation", _node.documentation() ? Json::Value(*_node.documentation()) : Json::nullValue),
+		make_pair("documentation", _node.documentation()->text() ? Json::Value(*_node.documentation()->text()) : Json::nullValue),
```

Comment:
I think we should append an AST node to the JSON output here. So this should not be the final solution.